scripts/generate/templates/Shim.mustache

```diff
@@ -15,7 +17,15 @@ This file is autogenerated, to keep it in sync with
 {-# LANGUAGE PatternSynonyms #-}
 {-# LANGUAGE RankNTypes #-}
 
-module Database.Persist.Monad.Shim where
+module Database.Persist.Monad.Shim (
+  module Database.Persist.Sql,
+  {{#functions}}
```

Comment:
Can the explicit export list here be replaced with `module Database.Persist.Monad.Shim`? I thought that used to be valid

Reply:
If you want to expose `fromAcquire` as well, then yes. Actually, omitting that function makes this a breaking change - I'm happy to do `module Database.Persist.Monad.Shim` instead.

Reply:
Ah actually never mind; it'd be better if fromAcquire wasnt exported. I'm fine with that breaking change